blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricsIndexHandler.java

```diff
@@ -24,7 +23,7 @@
     private DiscoveryIO discoveryHandle;
 
     public HttpMetricsIndexHandler() {
-        loadDiscoveryModule();
+        discoveryHandle = QueryDiscoveryModuleLoader.getDiscoveryInstance();
```

Comment:
@tilogaat I do not see the reason why we still cannot mock it, like can't we have a setter method like `public void setDiscoveryIO(DiscoveryIO handle)` which is visible only for testing with `@VisibleForTesting` annotation? But I would love to hear any counter-arguments on this...